pkg/policy/policy.go

```diff
@@ -22,12 +19,17 @@ type Policy struct {
 	ContFilter        *filters.BoolFilter
 	NewContFilter     *filters.BoolFilter
 	ContIDFilter      *filters.StringFilter
-	RetFilter         *filters.RetFilter
-	DataFilter        *filters.DataFilter
-	ScopeFilter       *filters.ScopeFilter
 	ProcessTreeFilter *filters.ProcessTreeFilter
 	BinaryFilter      *filters.BinaryFilter
 	Follow            bool
+	Rules             map[events.ID]RuleData
+}
+
+type RuleData struct {
+	EventID     events.ID
```

Comment:
Is this `EventID` necessary since we already have `events.ID` in Policy.Rules? I tried searching the code but couldn't find any place where it's needed - or maybe we'll need it in the future.

Reply:
My thought here was that a rule is composed from an event and its filters (soon action will be added as well), so I placed it here as well. This might be relevant in the future if we will want to pass a pointer to the Rule without giving a reference to the policy itself